system/settings/settings.xml

```diff
@@ -2561,40 +2561,13 @@
         </setting>
       </group>
       <group id="2">
-        <setting id="audiooutput.dspaddonsenabled" type="boolean" label="36441" help="36438">
```

Comment:
@FernetMenta @AlwinEsch 
This is really the way we should go. Automatically enable AudioDSP if the first add-on is enable, but my opinion is that it is to early to completly remove a button to enable/disable AudioDSP. Currently we have some issues with and it doesn't work properly on all platforms. I'm not a blocker, but I like to give the users the possibility to disable it completly until we have good support for all platforms.
According to @AlwinEsch we have disable all add-ons manually to disable AudioDSP completly.

What do you think about it?